apps/iotune/iotune.cc

```diff
@@ -376,7 +389,7 @@ class io_worker {
 
     io_worker(size_t buffer_size, std::chrono::duration<double> duration, std::unique_ptr<request_issuer> reqs, std::unique_ptr<position_generator> pos, std::vector<unsigned>& rates)
         : _buffer_size(buffer_size)
-        , _start_measuring(iotune_clock::now() + std::chrono::duration<double>(10ms))
+        , _start_measuring(iotune_clock::now() + std::chrono::duration<double>(warmup_period(duration)))
```

Comment:
This adds warmup_period(duration) for _every_ measurement and there are 4 of them -- random/sequential read/write. So ultimately, the total duration is increased 4 times the requested value, right?

Reply:
> This adds warmup_period(duration) for every measurement and there are 4 of them -- random/sequential read/write. 

Yes (and that is intentional)

> So ultimately, the total duration is increased 4 times the requested value, right? 

Not sure I fully understand but yes the total run time of iotune increases by four times the warmup period.